includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -513,6 +513,35 @@ protected function prepare_payment_information( $order ) {
 		return $payment_information;
 	}
 
+	/**
+	 * Upserts customer from user, name, and email.
+	 *
+	 * @param WP_User $user The user associated to customer.
+	 * @param string  $name The customer's name.
+	 * @param string  $email The customer's email.
+	 *
+	 * @return string|null Customer id.
+	 */
+	protected function upsert_customer( $user, $name, $email ) {
```

Comment:
And related to the comment below: we can add type hints for parameters here.

Reply:
OK. I also just realized `create_customer_for_user()` and `update_customer_for_user()` can throw an exception. Will handle that as well.